pulsar-io/couchbase/src/main/java/org/apache/pulsar/io/cb/CBSourceConfig.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.io.cb;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import org.apache.pulsar.io.core.annotations.FieldDoc;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Map;
+
+@Data
+@Accessors(chain = true)
+public class CBSourceConfig implements Serializable {
+  @FieldDoc(required = true, defaultValue = "",
+      help = "name connection to couchbase server")
+  private String connectionName;
+
+  @FieldDoc(required = true, defaultValue = "10000",
+      help = "Wait this milliseconds before failing when connecting to couchbase server")
+  private long connectionTimeOut;
```

Comment:
Default values should be initialized.

Reply:
> Default values should be initialized.

required = true, so can not initialized.